src/components/controls/time/Time.js

```diff
@@ -25,7 +25,7 @@ var Time = React.createClass({
      * @return {string}         Timestamp in the format of HH:MM:SS
      */
     formatTime(seconds) {
-        var date = new Date(null);
+        var date = new Date(Date.UTC(2000,1,1,0,0,0,0);
```

Comment:
Sorry for that, I had just cloned the project and when I forked it to make the pull request I didn't check. Now I'm facing some error while running npm install in the demo folder
![screen shot 2016-11-30 at 1 26 46 pm](https://cloud.githubusercontent.com/assets/3694228/20758353/ae65f6fa-b700-11e6-8048-2d2e89fa9473.png)